src/umpire/ResourceManager.cpp

```diff
@@ -163,6 +177,18 @@ ResourceManager::initialize()
     m_allocators.emplace_front(std::move(host_allocator));
   }
 
+  {
+    std::unique_ptr<strategy::AllocationStrategy>
+      allocator{
+        //util::wrap_allocator<strategy::AllocationTracker>(
+          registry.makeMemoryResource("NULL", getNextId())};
+
+    int id{allocator->getId()};
+    m_allocators_by_name["NULL"]  = allocator.get();
```

Comment:
`allocator.get()` could be assigned to a private variable of `ResourceManger` here.